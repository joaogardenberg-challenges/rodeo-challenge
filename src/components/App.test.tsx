import { render, screen } from '@testing-library/react'

jest.mock('./invoice/Invoice')

const App = require('./App').default

describe('App', () => {
  it('Renders the Invoice successfully', () => {
    const { container } = render(<App />)
    const invoice = screen.getByTestId('invoice')

    expect(invoice).toBeInTheDocument()
    expect(container).toMatchSnapshot()
  })
})
