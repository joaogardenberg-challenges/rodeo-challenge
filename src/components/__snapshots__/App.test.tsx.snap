// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Renders the Invoice successfully 1`] = `
<div>
  <div
    data-testid="invoice"
  />
</div>
`;
